extensions/resteasy-reactive/rest-client-reactive-jackson/runtime/src/main/java/io/quarkus/rest/client/reactive/jackson/runtime/serialisers/ClientJacksonMessageBodyReader.java

```diff
@@ -40,7 +39,7 @@ public Object readFrom(Class<Object> type, Type genericType, Annotation[] annota
         try {
             return super.readFrom(type, genericType, annotations, mediaType, httpHeaders, entityStream);
         } catch (StreamReadException | DatabindException e) {
-            throw new ClientWebApplicationException(e, Response.Status.BAD_REQUEST);
+            throw new WebApplicationException(e, Response.Status.BAD_REQUEST);
```

Comment:
As now this message body ready is used by the client, this change is necessary, if not, the test BadRequestNotPropagatedTestCase fails. Though, I think this change is correct.